### PR TITLE
iModel Evolution Tests Old Test Runners Ignore List Process

### PR DESCRIPTION
**Pull request by naveedkhan8067**
This upgrade will help to ignore tests in old test runners based on the test runners and their versions.

### CODE REVIEW

**Review by ColinKerr — APPROVED** [review]

File: iModelCore/iModelPlatform/Tests/DgnProject/Compatibility/CompatibilityRunner/RunPulledTestRunners.py [hunk 1]
```diff
@@ -11,19 +11,58 @@
...
+    ignoreList = []
+    allVersions = False
+
+    for item in rawList:
+        if item.count(".") <= 1:
+            print("Test format in the ignore list is not correct")
+            print("Please follow the correct format:")
+            print("-> TestRunnerName.version.TestFixtureName.TestName")
+            print("-> Example: testRunnerNuget_bim0200_x64.2019.2.22.1.CompatibilityTestFixture.ECSqlColumnInfoForAliases")
+            print("->    Note: Use `*` in place of TestRunnerName.version for applying it on all old test runners")
```
[the remark this answers is not in the document]
Comment (naveedkhan8067) [reply iModelCore/iModelPlatform/Tests/DgnProject/Compatibility/CompatibilityRunner/RunPulledTestRunners.py:32]: "not old test runner" means a test runner which is generated from the current code . 
"old test runners" are those which where generated at some point in time (in past) for example from bim0200dev stream and currently present as NuGet packages for usage.

[the remark this answers is not in the document]
Comment (pmconne) [reply iModelCore/iModelPlatform/Tests/DgnProject/Compatibility/CompatibilityRunner/RunPulledTestRunners.py:32]: But it appears to me that "*.Fixture.Test" skips it for **all** test runners, new and old; not "all **old** test runners" as described in output. Do I misunderstand?

[the remark this answers is not in the document]
Comment (naveedkhan8067) [reply iModelCore/iModelPlatform/Tests/DgnProject/Compatibility/CompatibilityRunner/RunPulledTestRunners.py:32]: No, this python script and the ignore list process is just for old test runners. Tests from the current code runs from a different part and have no impact from this ignore list.

[the remark this answers is not in the document]
Comment (pmconne) [reply iModelCore/iModelPlatform/Tests/DgnProject/Compatibility/CompatibilityRunner/RunPulledTestRunners.py:32]: Thanks for clarifying.

**Review by pmconne — APPROVED** [review]

**Pull request merged**